CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file. Do not remo
 
 ## Unreleased
 
+## 0.15.0
+
+#### :rocket: Enhancement
+* Other
```

Comment:
why other? It seems the effected modules detection has some issue

Reply:
It appears https://github.com/lerna/lerna-changelog only supports packages _in the packages folder_

I'm looking into alternatives now